app/src/components/modal/modal_airdrop/index.tsx

```diff
@@ -30,6 +39,7 @@ export const ModalAirdrop = (props: Props) => {
   const [checkAddress, setCheckAddress] = useState(false)
   const [isTransactionModalOpen, setIsTransactionModalOpen] = useState<boolean>(false)
   const [message, setMessage] = useState('')
+  const displayDaiBanner = balances.xDaiBalance.isZero()
```

Comment:
@pimato We should consider if we want to add dust checker. Instead of evaluation for strict zero. If Im not mistaken dust checker will evaluate to true if its less then 0.0001 so that means it will evaluate to true even if its zero...This should be checked though...